src/net.h

```diff
@@ -233,8 +233,8 @@ struct LocalServiceInfo {
 extern Mutex g_maplocalhost_mutex;
 extern std::map<CNetAddr, LocalServiceInfo> mapLocalHost GUARDED_BY(g_maplocalhost_mutex);
 
-extern const std::string NET_MESSAGE_COMMAND_OTHER;
-typedef std::map<std::string, uint64_t> mapMsgCmdSize; //command, total bytes
+extern const std::string NET_MESSAGE_TYPE_OTHER;
+typedef std::map<std::string, uint64_t> mapMsgTypeSize; //message, total bytes
```

Comment:
nit: Second commit `s/message/message type/`. Or:

```suggestion
using mapMsgTypeSize = std::map</* msg type */ std::string, /* total bytes */ uint64_t>;
```

Reply:
Thanks, @MarcoFalke, for taking another look. As per your suggestion, I was able to identify one more place in the second commit where your suggestion could be implemented.

```diff
-            // to prevent a memory DOS, only allow valid commands
+            // to prevent a memory DOS, only allow valid messages
```
But I opted against changing the above from `messages` to `message types`, as that would have decreased the sentence's logical soundness IMO.